packages/http/libraries/express/package.json

```
@@ -0,0 +1,77 @@
+{
+  "author": "Adrián Martínez Jiménez",
+  "bugs": {
+    "url": "https://github.com/inversify/monorepo/issues"
+  },
+  "description": "InversifyJs http express package",
+  "dependencies": {
+    "@inversifyjs/http-core": "workspace:*"
+  },
+  "devDependencies": {
+    "@stryker-mutator/core": "8.7.1",
+    "@stryker-mutator/vitest-runner": "8.7.1",
+    "@stryker-mutator/typescript-checker": "8.7.1",
+    "@types/express": "^4.17.21",
+    "@types/node": "22.13.4",
+    "@vitest/coverage-v8": "3.0.7",
+    "eslint": "9.20.1",
+    "prettier": "3.5.1",
+    "rimraf": "6.0.1",
+    "rollup": "4.34.8",
+    "ts-loader": "9.5.2",
+    "tslib": "2.8.1",
+    "typescript": "5.7.3",
+    "vitest": "3.0.7"
+  },
+  "devEngines": {
+    "node": "^22.10.2",
+    "pnpm": "^9.12.1"
+  },
+  "homepage": "https://inversify.io",
+  "keywords": [
+    "dependency injection",
+    "dependency inversion",
+    "di",
+    "inversion of control container",
+    "ioc",
+    "javascript",
+    "node",
+    "typescript"
+  ],
+  "license": "MIT",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
+  "name": "@inversifyjs/http-express",
```

```suggestion
  "name": "@inversifyjs/http-express",
  "private": "true",
```
If it's not ready to be deployed, let's mark it as private for now.

`.changeset/config.json` should be updated so this package is ignored